script.js

```diff
@@ -17,4 +17,130 @@ document.querySelectorAll('.nav-list').forEach((e) => e.addEventListener('click'
+    technologies: ['html', 'Ruby on rails', 'css'],
+    technologies2: ['css', 'html', 'Bootstrap', 'Ruby'],
+    liveVersion: '#',
+    source: '#',
+  },
+  {
+    id: 4,
+    name: 'Multi - Post stories4',
+    featuredImage: 'placeholder.png',
+    featuredImage2: 'modal.png',
+    description:
+    'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.',
+    desc:
+    'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting  ever since the 1500s, when an unknown printer took a galley of type veris lapoa todoe.',
+    technologies: ['html', 'Ruby on rails', 'css'],
+    technologies2: ['css', 'html', 'Bootstrap', 'Ruby'],
+    liveVersion: '#',
+    source: '#',
+  },
+];
+
+const mainBody = document.querySelector('.main-body');
+const windowPop = document.querySelector('.window-popup');
+const wrapper = document.querySelector('.wrapper');
+
+const mainContainer = projects.map((project) => `<div
+class="projects-container hide-in-desktop show">
+<div
+    class="image-placeholder hide-in-desktop show">
+    <img src="IMG1/${project.featuredImage}"
+        alt="${project.name}">
+</div>
+<div
+    class="content hide-in-desktop show">
+    <div class="text-area">
+        <h2>${project.name}</h2>
+        <p>${project.description}</p>
+    </div>
+    <ul class="list-tags">
+        ${project.technologies2.map((technology) => `<li class="border-right">${technology}</li>`).join('')}
+    </ul>
+    <button type="button" onclick="showModal(${project.id})"
+        class="myprojects">See
+        projects</button>
+</div>
+</div>`).join('');
+
+// append the main-container to the main-body container
+mainBody.innerHTML += mainContainer;
+
+// create the modal pop up
+
+const windowContent = (modal) => {
+  const template = document.createElement('template');
+  template.innerHTML = ` <div class="heading">
+<h2>${modal.name}</h2>
+<button class="close" onclick="closeModal()" >&times</button>
+</div>
+<img src="IMG1/${modal.featuredImage2}" alt="${modal.name}">
+<p>${modal.desc}</p>
+<ul class="tags">
+${modal.technologies.map((technology) => `<li>${technology}</li>`).join('')}
+</ul>
+<div class="click-me">
+<button type="button" class="see-live">See live</button>
+<button type="button" class="source">See source</button>
```

Comment:
- [ ] **Required:** Great improvement made to the project. 👏👏👏 However, as the previous reviewer pointed out, all elements in the popup modal are centred to the modal. Kindly canter the buttons.

Your Design | Template
:---: | :---:
![image](https://user-images.githubusercontent.com/27709832/158036279-e97d7349-0cd8-4101-bec5-b58b17977338.png) | ![image](https://user-images.githubusercontent.com/27709832/158036295-756c7dd0-d74c-448e-a202-44837ad012c4.png)

- [ ] **Optional:** I also strongly suggest that you reduce the `width` and `height` of the modal and set the[ truncate the content](https://css-tricks.com/snippets/css/truncate-string-with-ellipsis/) so it can be uniform.